go/src/vendors/sonic/client/dbclient/counter_repository_test.go

```diff
@@ -0,0 +1,243 @@
+package dbclient
+
```

Comment:
Can you please add coverage % in PR description and also share the coverage.html ?

Reply:
% coverage is already added into summary. Have attached the coverage file here. https://github.com/renukamanavalan/sonic-device-health/pull/8#issuecomment-1512675723